src/git.ts

```diff
@@ -51,6 +51,15 @@ export class Repo {
     return this.gitExec(['diff', '-U0']);
   }
 
+  /**
+   * Determine the worktree root of this repository.
+   *
+   * @returns The worktree root
```

Comment:
You can reword this to `Fulfills with the worktree root`. It would be more correct plus you have done a similar thing for other methods in this class.

Reply:
Right, nice catch!